src/components/common/NameRequestInfo.vue

```diff
@@ -0,0 +1,161 @@
+          </li>
+        </ul>
+      </v-col>
+      <v-col id="name-request-applicant-info" cols="8">
+        <ul>
+          <li><label>
+          <strong>Name Request Applicant</strong>
+        </label></li>
+          <li>Name: {{ applicantName() }}</li>
+          <li>Address: {{ applicantAddress() }}</li>
+          <li>Email: {{ nameRequestApplicant.emailAddress }}</li>
+          <li>Phone: {{ nameRequestApplicant.phoneNumber | VMask('(###) ###-####')}}</li>
+        </ul>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins, Vue, Prop } from 'vue-property-decorator'
+import { getName } from 'country-list'
+import { VueMaskFilter } from 'v-mask'
+
+// Enums
+import { NameRequestStates } from '@/enums'
+
+// Interfaces
+import { NameRequestIF, NameRequestDetailsIF, NameRequestApplicantIF } from '@/interfaces'
+
+// Mixins
+import { CommonMixin, DateMixin, NamexRequestMixin } from '@/mixins'
+
+@Component({
+  filters: { 'VMask': VueMaskFilter }
+})
+export default class NameRequestInfo extends Mixins(CommonMixin, DateMixin, NamexRequestMixin) {
+  // Template Enums
+  readonly NameRequestStates = NameRequestStates
+
+  @Prop()
+  private nameRequest: any
+
+  private parsedNameRequest: NameRequestIF
+
+  private nameRequestDetails:NameRequestDetailsIF
+
+  private nameRequestApplicant: NameRequestApplicantIF
+
+  private created () {
+    this.parsedNameRequest = this.generateNameRequestState(this.nameRequest, null)
+    this.nameRequestDetails = this.parsedNameRequest.details
+    this.nameRequestApplicant = this.parsedNameRequest.applicant
+  }
+
+  /** The entity title  */
+  private entityTypeDescription () :string {
+    const entityType = this.parsedNameRequest.entityType
+    if (entityType === 'BC') {
+      return 'Benefit Company'
+    } else if (entityType === 'CP') {
+      return 'Cooperative Association'
+    }
+    return ''
+  }
+
+  /** The request type */
+  private requestType () : string {
+    return 'New Business'
+  }
+
+  /** Return formatted expiration date */
+  private formattedExpirationDate () : string {
+    return this.toReadableDate(this.nameRequestDetails.expirationDate)
+  }
+
+  /** Return consent received string by checking if conditional and if consent has been received */
+  private conditionConsent (): string {
+    if (this.nameRequestDetails.status === NameRequestStates.CONDITIONAL) {
+      return this.nameRequestDetails.consentFlag === 'R' ? 'Received' : 'Not Received'
+    } else {
+      return 'Not Required'
+    }
+  }
+
+  /** Return formatted applicant name */
+  private applicantName () : string {
+    let name = this.nameRequestApplicant.firstName
+    if (this.nameRequestApplicant.middleName) {
+      name = `${name} ${this.nameRequestApplicant.middleName} ${this.nameRequestApplicant.lastName}`
+    } else {
+      name = `${name} ${this.nameRequestApplicant.lastName}`
```

Comment:
Did you know you could contact string like this?
`name += this.nameRequestApplicant.lastName`

Reply:
> Did you know you could concat strings like this?
> `name += this.nameRequestApplicant.lastName`

Reply:
Ofcourse I know :). Then you will have to take care of adding spaces in between. This looks better to me and I have taken this section as it is from Create Business

Reply:
Haha, awesome :)